### PR TITLE
descriptor.md: of size->and size of

### PR DESCRIPTION
**Pull request by TrumanLing**
Signed-off-by: Ling FaKe <lingfake@huawei.com>

### CODE REVIEW

File: descriptor.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Implementations MUST implement SHA-256 digest verification for use in descriptor
 
 ## Examples
 
-The following example describes a [_Manifest_](manifest.md#image-manifest) with a content identifier of "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270", of size 7682 bytes:
+The following example describes a [_Manifest_](manifest.md#image-manifest) with a content identifier of "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270",  and size of 7682 bytes:
```
Comment (wking) [inline descriptor.md]: I agree that this line could be improved, but how about:

> … with a digest of "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270" and a size of 7682 bytes:

**Comment by TrumanLing** [comment]
@wking good advice. I have updated it, PTAL.

File: descriptor.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ Implementations MUST implement SHA-256 digest verification for use in descriptor
 
 ## Examples
 
-The following example describes a [_Manifest_](manifest.md#image-manifest) with a content identifier of "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270", of size 7682 bytes:
+The following example describes a [_Manifest_](manifest.md#image-manifest) with a content identifier of "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270",  and a size of 7682 bytes:
```
Comment (wking) [inline descriptor.md]: You have “,  ” (a comma and two space) where [I think we want][1] a single space.

[1]: https://github.com/opencontainers/image-spec/pull/426#discussion_r85781936

Comment (TrumanLing) [inline descriptor.md]: @wking got it.

**Comment by TrumanLing** [comment]
@wking updated it, PTAL.

**Comment by wking** [comment]
7f1a5912 still has the comma I don't think we need, but it's down to
one space, and that's good enough for me.

**Comment by TrumanLing** [comment]
@wking I missed something and updated again, PTAL.

**Comment by wking** [comment]
454e5c649 looks good to me :).

**Comment by jonboulle** [comment]
lgtm

**Comment by vbatts** [comment]
LGTM

**Pull request merged**